extension/package.json

```diff
@@ -34,6 +34,12 @@
     "workspaceContains:node_modules/@angular/cli/bin/ng",
     "onCommand:extension.componize"
   ],
+  "dependencies": {
```

Comment:
The dependencies should be in the main package.json (in the root folder) then you should put these as peerDependencies

This will avoid duplication and sync the package versions across the schematics and extension project